src/olympia/addons/serializers.py

```diff
@@ -144,11 +145,18 @@ class Meta:
         fields = ('id', 'name', 'url')
 
 
-class SimpleVersionSerializer(serializers.ModelSerializer):
+class MinimalVersionSerializer(serializers.ModelSerializer):
```

Comment:
Minimal is the new Simple! 😆

Reply:
Can't wait for the new "ExtremelyMinimalVersionSerializer" eh...

Reply:
NanoVersionSerializer